<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckingStatementInfoWizardPage</class>
 <widget class="QWizardPage" name="CheckingStatementInfoWizardPage">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>34</y>
     <width>577</width>
     <height>388</height>
    </rect>
   </property>
   <attribute name="title">
    <string>Statement Information</string>
   </attribute>
   <layout class="QVBoxLayout">
    <item>
     <widget class="QLabel" name="m_enterInformationLabel" native="true">
      <property name="text" stdset="0">
       <string></string>
      </property>
      <property name="wordWrap">
        <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="Spacer4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>50</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <spacer name="Spacer3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>42</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="TextLabel3">
          <property name="text">
           <string>Statement date:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KMyMoneyDateInput" name="m_statementDate"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="TextLabel2">
          <property name="text">
           <string>Starting balance of this statement:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
          <widget class="AmountEdit" name="m_previousBalance">
            <property name="enabled">
              <bool>false</bool>
            </property>
          </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="TextLabel4">
          <property name="text">
           <string>Ending balance of this statement:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="AmountEdit" name="m_endingBalance"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="Spacer2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>46</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="Spacer5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>16</width>
        <height>87</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="m_oldestTransactionDate" />
    </item>
    <item>
     <widget class="QLabel" name="m_lastStatementDate">
      <property name="text">
       <string>1</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
 </widget>
 <resources/>
 <connections/>
 <customwidgets>
   <customwidget>
     <class>AmountEdit</class>
     <extends>QLineEdit</extends>
     <header location="global">amountedit.h</header>
   </customwidget>		 
 </customwidgets>
 <tabstops>
 </tabstops>
 <buttongroups>
 </buttongroups>
</ui>
